setup.cfg

```diff
@@ -18,6 +18,7 @@ addopts =
   -p no:logging
   --color=yes
   --doctest-glob=*.rst
+  --doctest-modules
```

Comment:
this change are the ones which are blocked https://github.com/coala/corobo/pull/616

please remove, and make sure CI is green